lib/tasks/rails_db_views_tasks.rake

```diff
@@ -25,28 +25,17 @@ namespace :db do
 
     creator.drop_views
   end
+end
 
-  task :migrate => :environment do
-    Rake::Task['db:create_views'].invoke
-  end
-
-  task :rollback => :environment do
-    Rake::Task['db:create_views'].invoke
-  end
+# prepend "db" namespace
```

Comment:
Take a look at https://github.com/wcmonty/rails_db_views/pull/3 when you can which hopefully can be merged right away -- and then if you want to just kill this PR, I definitely understand, I was just sharing the thoughts that led to this refactor.

I do think that if we ever see ourselves contributing to the original gem, the most important things would be:

1. the removal of rake-hooks because of its flaws -- tbh I forget *exactly* what the issue is, I just remember that it isn't good... I think it was >a year ago when you and I were looking at that.
2. The additional rake tasks to augment.
3. The refactor in this PR

But mostly I'm cool with getting away from this gem and not worrying about contributing heh.